mypy/test/data.py

```diff
@@ -556,6 +556,9 @@ def repr_failure(self, excinfo: Any) -> str:
 
 
 class DataSuite:
+    def __init__(self, *, update_data: bool = False) -> None:
```

Comment:
The only class that actually defines a nontrivial `__init__` is `ASTMergeSuite` (and I'm not sure it should, since it does not receive arguments and does not seem to have any mutable state).
May I simply remove the empty `__init__` definitions?